content/docs/2.0/concepts/scaling-jobs.md

```diff
@@ -40,6 +40,9 @@ spec:
   failedJobsHistoryLimit: 5                 # Optional. Default: 100. How many failed jobs should be kept.
   envSourceContainerName: {container-name}  # Optional. Default: .spec.JobTargetRef.template.spec.containers[0]
   maxReplicaCount: 100                      # Optional. Default: 100
+  scalingStrategy: "custom"                 # Optional. Default: default Which ScalingStrategy to use. 
```

Comment:
I'd change this as mentioned in https://github.com/kedacore/keda/pull/1227#discussion_r500897086

Reply:
And fix to comment please:

`# Optional. Default: default. Which Scaling Strategy to use. `
(added . behind `default` and used space in `Scaling Strategy` )

Reply:
Done